src/odemis/driver/smaract.py

```diff
@@ -1195,11 +1195,7 @@ def __init__(self, name, role, locator, axes, ref_on_init=False, linear_speed=0.
         pos_deactive_after_ref (bool): if True, will move to the deactive position
             defined in metadata after referencing
         """
-        if locator != "fake":
-            self.core = MC_5DOF_DLL()
-        else:
-            self.core = FakeMC_5DOF_DLL()
-
+        self.core = MC_5DOF_DLL() if locator != "fake" else FakeMC_5DOF_DLL()
```

Comment:
It's probably a matter of taste, and I a bit old-school, but I don't think this helps reading.